frontend/src/pages/secret-manager/SecretDashboardPage/components/ActionBar/CreateDynamicSecretForm/SqlDatabaseInputForm.tsx

```diff
@@ -23,6 +23,25 @@ import { useWorkspace } from "@app/context";
 import { gatewaysQueryKeys, useCreateDynamicSecret } from "@app/hooks/api";
 import { DynamicSecretProviders, SqlProviders } from "@app/hooks/api/dynamicSecret/types";
 
+const passwordRequirementsSchema = z.object({
+  minLength: z.number().min(1).max(100),
+  maxLength: z.number().min(1).max(100),
+  required: z.object({
+    lowercase: z.number().min(0),
+    uppercase: z.number().min(0),
+    digits: z.number().min(0),
+    symbols: z.number().min(0)
+  }),
```

Comment:
I think we should add validation here to ensure that the total of required fields does not exceed the length?

Reply:
done